### PR TITLE
Add david-dm dependency status badges to README

### PR DESCRIPTION
**Pull request by cflynn07**
These badges are nice to have

### CODE REVIEW

**Comment by bkendall** [comment]
LGTM! Thanks :)

**Comment by bkendall** [comment]
Is the first link broken, or is it David or my iPad internet?

**Comment by cflynn07** [comment]
Think it might be your ipad internet
screen shot 2015-10-11 at 11 08 12 am

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ![101](http://i.imgur.com/MFrmMt6.png)
 ===
-[![NPM](https://nodei.co/npm/101.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/101/)
+[![NPM](https://nodei.co/npm/101.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/101/)  
+
 [![Build Status](https://travis-ci.org/tjmehta/101.svg?branch=master)](https://travis-ci.org/tjmehta/101)
 [![Coverage Status](https://coveralls.io/repos/tjmehta/101/badge.png)](https://coveralls.io/r/tjmehta/101)
+[![Dependency Status](https://david-dm.org/tjmehta/101.svg)](https://david-dm.org/tjmheta/101)
```
Comment (bkendall) [inline README.md]: Ah ha! There is a typo in here. You swapped the h and the e in his username in the link

**Comment by tjmehta** [comment]
:+1:

**Comment by bkendall** [comment]
@tjmehta that link is broken... I'll make another PR...

**Pull request merged**